__tests__/utils/providers.tsx

```diff
@@ -21,10 +20,6 @@ export type Props = {
 export const Wrapper: React.FC<Props> = ({ children, reduxState = initialRootState(), route = '/' }) => {
   window.history.pushState({}, 'Test page', route)
 
-  posthog.init('__test__posthog__token', {
```

Comment:
Last time, I removed something in the wrapper that was breaking our test, it was something like : `server` and it was doing something with outside HTTP calls. I did not know why we used it so far because everything was working the same. And as it was broken, and our tests were broken too, I removed this server thing.
Then, few days later, I had a problem with posthog. I then removed it because we don't need it nowhere in our tests except on the Login component. I removed it from our global wrapper and added it in the only component it was required. Inside the `page-logout-feature.spec.tsx`